src/components/news-item.vue

```diff
@@ -24,6 +24,25 @@
     <template v-else-if="is('Picture')">
       <ion-img class="item-img" :src="imageUrl" />
     </template>
+    <template v-else>
+      <ion-card class="post-text ion-padding">
+        {{ $t("activity.unknownType") }}
+        <ion-grid>
+          <ion-row>
+            <ion-col>
+              <ion-button fill="outline" :href="androidLink">{{
+                $t("appPage.button.android")
+              }}</ion-button>
+            </ion-col>
+            <ion-col>
+              <ion-button fill="outline" :href="iosLink">{{
+                $t("appPage.button.iOS")
+              }}</ion-button>
+            </ion-col>
+          </ion-row>
+        </ion-grid>
```

Comment:
Any way we can detect which one this is? We already do something similar for the install-popup, check how we do it there? And, though secondary, we do support the html-version too and should probably add a refresh-button as fallback.

Reply:
Could you point me to where you already do that? I could only find this, so I did it similarly: https://github.com/TeamFranka/affinity/blob/ac882f931e3599104fef759bd3cfbdb5c580d822/src/views/App.vue#L11-L26

Reply:
Here https://github.com/TeamFranka/affinity/blob/ac882f931e3599104fef759bd3cfbdb5c580d822/src/config/App.vue#L164-L225